x-pack/platform/plugins/private/painless_lab/test/scout/api/tests/execute_api.spec.ts

```diff
@@ -47,3 +51,96 @@ apiTest.describe(
     });
   }
 );
+
+apiTest.describe(
```

Comment:
I've added new stateful-only tests that use both Elasticsearch role descriptors and Kibana role descriptors.

Reply:
Do we want to set up an ESLint rule that flags Scout test files with two test suites? I can create an issue.

Reply:
I moved the second test suite to a new file called `execute_api_custom_cluster_privileges.spec.ts` in [fa99c6b](https://github.com/elastic/kibana/pull/243187/commits/fa99c6bde8693ff6e99c5fd399a4569494f52523).